src/bindings/go-elektra/kdb/kdb.go

```diff
@@ -0,0 +1,160 @@
+package kdb
+
+// #cgo pkg-config: elektra
```

Comment:
The immediate cause of the space issue is probably this. However, this line is not problematic and supports spaces properly (https://github.com/golang/go/issues/16455), as long as the pkg-config file is correct.

However, the bigger issue here is that this will not work, even if the `PKG_CONFIG_PATH` was set better in CMake. The issue is that `pkg-config` only finds an installed version of Elektra, it the pkgconfig files will never work properly for the version still being built. We'd have to generate special variants of the pkgconfig files and use those, but then when building for release packages we'd have to switch back to the normal ones.

Reply:
> However, the bigger issue here is that this will not work, even if the PKG_CONFIG_PATH was set better in CMake. The issue is that pkg-config only finds an installed version of Elektra, it the pkgconfig files will never work properly for the version still being built. We'd have to generate special variants of the pkgconfig files and use those, but then when building for release packages we'd have to switch back to the normal ones.

As I am not an expert on this, I unfortunately didn't understand what you mean :smile:

Reply:
> The immediate cause of the space issue is probably this. However, this line is not problematic and supports spaces properly ([golang/go#16455](https://github.com/golang/go/issues/16455)), as long as the pkg-config file is correct.

The error messages was a different one. This worked correctly but it then could not find other includes which were located under `....../src/includes`. In the specific case (on jenkins) it was `directory/src/includes` it could not found. The `build ` was omitted from the folder name and it therefore did not find the correct includes.

Reply:
While trying to find the actual problem I also stumbled upon this https://github.com/golang/go/issues/16455.